xbmc/peripherals/devices/PeripheralCecAdapter.cpp

```diff
@@ -1187,7 +1191,7 @@ void CPeripheralCecAdapter::CecSourceActivated(void *cbParam, const CEC::cec_log
   if (activated == 1)
     g_application.WakeUpScreenSaverAndDPMS();
 
-  if (adapter->GetSettingBool("pause_playback_on_deactivate"))
+  if (adapter->GetSettingBool("pause_playback_on_deactivate") != LOCALISED_ID_NO_ACTION)
```

Comment:
Can `GetSettingBool` really be used now that there are more than two possible values (and they're integers)?